app/services/search/query_builders/chat_channel_membership.rb

```diff
@@ -0,0 +1,77 @@
+module Search
+  module QueryBuilders
+    class ChatChannelMembership
+      FILTER_KEYS = %i[
```

Comment:
These are all keys that [we currently use as filters](https://github.com/thepracticaldev/dev.to/blob/master/app/models/chat_channel_membership.rb#L25), I set them up in an array so we can easily add more when we want to.